source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -120,11 +120,12 @@ GLSSharpNavierStokesSolver<dim>::generate_cut_cells_map()
           // this value will never be used.
 
           cell->get_dof_indices(local_dof_indices);
-
+          unsigned int p_count = 0;
```

Comment:
What is p_count? Particle count? number_of_particle in cell? Please be more verbose iwth variable name.